src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -900,6 +900,197 @@ public RegexOtherEscapeFormatDefinition()
         }
         #endregion
 
+        #region JSON
+
+        [Export(typeof(EditorFormatDefinition))]
+        [ClassificationType(ClassificationTypeNames = ClassificationTypeNames.JsonComment)]
+        [Name(ClassificationTypeNames.JsonComment)]
+        [Order(After = ClassificationTypeNames.StringLiteral)]
+        [Order(After = ClassificationTypeNames.VerbatimStringLiteral)]
+        [UserVisible(true)]
+        [ExcludeFromCodeCoverage]
+        private class JsonCommentFormatDefinition : ClassificationFormatDefinition
```

Comment:
❔ Are we reusing the classifications for JSON files?

Reply:
We are not.  These are the classifications Kasey and I came up with when doing the original featurework.  In particular, like with regex, we wanted the values to look good in the context of surrounding c# code.  our trials using the actual json classification values indicated they didn't look good when surrounded by C#.

Reply:
Do these colors work for both the light and dark editor themes? We may want to add entries to our VS2017 and VS2019 color scheme files for them - https://github.com/dotnet/roslyn/tree/main/src/VisualStudio/Core/Def/ColorSchemes. I would be happy to add these in a follow up PR once this goes in.

Reply:
> Do these colors work for both the light and dark editor themes? 

the colors actually match the regex colors (that was intentional when we did this original work).  So whatever we had to do to make regex work would apply here.  

> We may want to add entries to our VS2017 and VS2019 color scheme files for them -

I'd be happy to get as much as help as possible there :)

Reply:
hrmmmm.  i switched to hex, and they dont' match the regex values.  looks liek things changed at one point.  will try to rematch the values up now.

Reply:
ok.  joey helped me through this!